Tasks/VSMobileCenterTest/task.json

```diff
@@ -207,6 +208,26 @@
             "groupName": "prepare"
         },
         {
+            "name":"xcuitestBuildDir",
+            "type":"string",
+            "label":"Build Directory",
+            "defaultValue":"",
+            "visibleRule":"enablePrepare = true && framework = xcuitest",
+            "required": false,
+            "helpMarkDown": "Path to the build output directory (usually  <project>/Build/Products/Debug-iphoneos)",
```

Comment:
I copied this from the mobile-center CLI help and it had extra space in it. I will update.